language/move-prover/tests/sources/functional/loops.move

```diff
@@ -0,0 +1,86 @@
+module VerifyLoops {
+
+    spec module {
+        pragma verify=false;
+    }
+
+    // ----------------------
+    // `aborts_if` statements
+    // ----------------------
+
+    public fun iter10_missing_inc_spec1() {
+        let i = 0;
+        while (i <= 10) { // an infinite loop
+            if (i > 10) abort 10;
+        }
+    }
+    spec fun iter10_missing_inc_spec1 { // TODO: What is the expected output for this spec?
+        aborts_if false;
```

Comment:
I think this will verify because aborts_if is a postcondition and Move Prover should be able to prove that control will never reach the procedure exit.

Reply:
I see. I noticed that Prover proved both `iter10_missing_inc_spec1` and `iter10_missing_inc_spec2`. I just wondered what output from Prover we desire in these cases.